sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_set/feature.py

```diff
@@ -6,15 +6,29 @@
 
 from typing import Dict, Optional
 
-from azure.ai.ml._restclient.v2023_04_01_preview.models import Feature as RestFeature, FeatureProperties
-
-from azure.ai.ml.entities._mixins import RestTranslatableMixin
-from azure.ai.ml.entities._feature_store_entity.data_column_type import DataColumnType
+from azure.ai.ml._restclient.v2023_04_01_preview.models import Feature as RestFeature
+from azure.ai.ml._restclient.v2023_04_01_preview.models import FeatureProperties
 from azure.ai.ml._utils._experimental import experimental
+from azure.ai.ml.entities._feature_store_entity.data_column_type import DataColumnType
+from azure.ai.ml.entities._mixins import RestTranslatableMixin
 
 
 @experimental
 class Feature(RestTranslatableMixin):
+    """Feature
+
+    :param name: The name of the feature resource.
```

Comment:
Remove "resource". Feature is not a resource.

Reply:
fixed